libraries/botbuilder-dialogs-adaptive/src/input/confirmInput.ts

```diff
@@ -218,4 +216,26 @@ export class ConfirmInput extends InputDialog implements ConfirmInputConfigurati
 
         return culture;
     }
+
+    private async getConfirmChoices(dc: DialogContext, defaults: (string | Choice)[]): Promise<ChoiceSet> {
```

Comment:
Once you drop the `await` you can remove `async` here as you're returning a Promise already.